about.html

```diff
@@ -76,10 +76,10 @@ <h2>
       <div class="row justify-content-center text-center">
         <div class="col col-lg-8">
           <h2>
-            Problems can be redefined with novel solutions
+            Redefine problems with novel solutions
           </h2>
           <p>
-            Vowpal Wabbit is structured around the principle of reducing problems to known solutions. When you can structure your problem as a class of problem that has been solved before then you can leverage existing technology. This is how you think about using VW to solve your problems, but it is also how VW works internally. Each reduction reduces to something that has been solved before and in this way we can reuse efficient algorithms that have stood the test of time.
+            Vowpal Wabbit is built on the principle of reducing problems to a class of problem with a known solution using fast, efficient, and proven algorithms.

```

Comment:
Oversimplification makes this statement hollow.

Reply:
It says the same thing in a concise manner. Would it help to expand with some bullets or another focused statement so we can link out to supporting content? 

Original is clunky and redundant. If reader wants more they jump to that content from this page.